src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/PrelinkTests.cs

```diff
@@ -36,7 +36,7 @@ public void Prelink_NullMethod_ThrowsArgumentNullException()
         [Fact]
         public void Prelink_NonRuntimeMethod_ThrowsArgumentException()
         {
-            AssertExtensions.Throws<ArgumentException>(null, () => Marshal.Prelink(new NonRuntimeMethodInfo()));
+            AssertExtensions.Throws<ArgumentException>("m", () => Marshal.Prelink(new NonRuntimeMethodInfo()));
```

Comment:
Is this going to pass CI without the coreclr update that's coming in?

Reply:
@stephentoub Nope. This is going to fail. I should add the `ActiveIssue`